### PR TITLE
documentation: Fix typo in bearlib/__init__.py

### PR DESCRIPTION
**Pull request by siddjai**
Changed nondeterministic to non-deterministic

Closes https://github.com/coala/coala/issues/2838

### CODE REVIEW

**Comment by adtac** [comment]
Please remove the merge commit for review. You can find more [here](http://docs.coala.io/en/latest/Developers/Git_Basics.html). Marking as WIP till then.

File: coalib/bearlib/__init__.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def deprecate_settings(**depr_args):
      ['old']
 
      You cannot deprecate an already deprecated setting. Don't try. It will
-     introduce nondeterministic errors to your program.
+     introduce non-deterministic errors to your program.
```
Comment (raul1991) [inline coalib/bearlib/__init__.py]: > " errors to your program" -> "errors in your program".


Change it.

**Comment by Nosferatul** [comment]
Hello! You should rebase because `out-of-date` branch

Rebasing
As people work on coala new commits will be added. This will result in your local fork going out of sync with the remote repository. To sync your changes with the remote repository run the following commands in the desired branch:

`$ git fetch origin`
`$ git rebase origin/master`
This will fetch the commits from the remote repository and will merge it into the branch where you are currently working, and move all of the local commits that are ahead of the rebased branch to the top of the history on that branch.

Note

After following these instructions when you try to push to remote you may get fast-forwarding error. If that is the case, then you will have to force push since you are attempting to rewrite the git commit history. To do that append the --force argument in the push command:

`$ git push myfork --force`

Warning: Never force-push on the master branch, or any branch not owned by you.
To verify whether you have rebased correctly, go to the web page of the branch in your fork. If it says your branch is n commits behind coala:master (or whichever repo you are contributing to), then you haven’t correctly rebased yet. Otherwise, you’re good to go!

**Comment by Nosferatul** [comment]
You have unwanted commits. You should squash them

Squashing your commits
It’s possible that you have more than one commit and you want them to be squashed into a single commit. You can take your series of commits and squash them down into a single commit with the interactive rebasing tool. To squash your commits run the following command:

`$ git rebase -i master`
Note

master is the SHA1 hash of the commit before which you want to squash all the commits and make sure that rebase is done onto master branch.
An editor will be fired up with all the commits in your current branch (ignoring merge commits), which come after the given commit. Keep the first one as “pick” and on the second and subsequent commits with “squash”. After saving, another editor will be fired up with all the messages of commits which you want to squash. Clean up all the messages and add a new message to be displayed for the single commit.

**Comment by Nosferatul** [comment]
Also, your commit shortlog has to many chars. Try: `__init__.py: Fix typo`. Marked as WIP.

**Comment by AbdealiJK** [comment]
@siddjai In it's current state, this cannot be merged as you have 4 "Merge" commits. Do a rebase, not a merge.
Do you plan to fix this ?

**Comment by sils** [comment]
closing due to inactivity

**Pull request closed without merging**